### PR TITLE
Fixed #52 Replace http.flow with eventManager

### LINKED ISSUE #52: change httpflow with event manager
Only Http? I'm not sure that this is the best name for this concept..

I call it with a classic name EventManager

### PR DESCRIPTION
**Pull request by gianarb**

### CODE REVIEW

**Comment by EmanueleMinotto** [comment]
I think you should choose a naming strategy, I see both lowercase with dots and camel case :\

**Comment by gianarb** [comment]
heh good feedback :D suggestions? :)

**Comment by EmanueleMinotto** [comment]
use the [snake_case](https://en.wikipedia.org/wiki/Snake_case), in this way you'll be able to distinguish immediately between code invocations and other concepts

``` php
$object->get('doMethod'); // invokes ::doMethod() on the internal object
$object->get('event_manager'); // gets the implementation of the event manager
```

File: docs/flow.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ require_once "./vendor/autoload.php";
 
 $app = new \GianArb\Penny\App();
 
-$app->getContainer()->get("http.flow")->attach("*", function ($event) {
+$app->getContainer()->get("eventManager")->attach("*", function ($event) {
```
Comment (samsonasik) [inline docs/flow.md]: event_manager too

Comment (gianarb) [inline docs/flow.md]: Thanks! :)

**Comment by gianarb** [comment]
Thanks at all! :D

**Pull request merged**